com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -158,6 +158,12 @@ public enum FrameSettingsField
         /// <summary>When enabled, Cameras using these Frame Settings render water surfaces.</summary>
         [FrameSettingsField(0, autoName: Water, customOrderInGroup: 106, tooltip: "When enabled, Cameras using these Frame Settings render water surfaces.")]
         Water = 99,
+        /// <summary>When enabled, HDRP accounts for asymmetry in the projection matrix.</summary>
+        [FrameSettingsField(0, displayedName: "Asymmetric Projection", customOrderInGroup: 107, tooltip: "When enabled HDRP will correctly render using asymmetric projection matrices, even when physical properties are used.")]
+        AsymmetricProjection = 78,
+        /// <summary>When enabled, HDRP evaluates post effects using transformed screen space coordinates.</summary>
+        [FrameSettingsField(0, displayedName: "Screen Coordinates Override", customOrderInGroup: 108, tooltip: "When enabled HDRP will use Screen Coordinate Override for post processing and custom passes.")]
```

Comment:
Statement is not self explaining. We should say that this allow cluster display as when reading current message, I still have no idea what is the usage.

Also please add documentation inside the doc folder (i.e the .md file for framesettings)

Reply:
https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md

Reply:
thanks, on a side note it seems to me some checkboxes are not documented, ex: Virtual Texturing, Water, ...

Reply:
thanks for reporting non documented checkbox, will poke people